lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -0,0 +1,110 @@
+;(function (define) {
+    'use strict';
+
+    define([
+        'backbone',
+        'underscore',
+        'moment',
+        'datatables',
+        'support/js/views/enrollment_modal',
+        'support/js/collections/enrollment',
+        'text!support/templates/enrollment.underscore'
+    ], function (Backbone, _, moment, dataTables, EnrollmentModal, EnrollmentCollection, enrollmentTemplate) {
+        return Backbone.View.extend({
+
+            ENROLLMENT_CHANGE_REASONS: [
```

Comment:
Hm, that's a good question. They need to be recorded un-translated, but it would make sense to display them translated.